app/storage/encrypted_questionnaire_storage.py

```diff
@@ -1,14 +1,13 @@
 import hashlib
 
-from app import settings
 from app.cryptography.jwe_decryption import JWEDirDecrypter
 from app.cryptography.jwe_encryption import JWEDirEncrypter
 from app.storage.questionnaire_storage import QuestionnaireStorage
 from app.utilities.strings import to_bytes
 from app.utilities.strings import to_str
 
 
-def generate_key(user_id, user_ik, pepper=settings.EQ_SERVER_SIDE_STORAGE_ENCRYPTION_USER_PEPPER):
+def generate_key(user_id, user_ik, pepper=''):
```

Comment:
Probably shouldn't default the pepper values here. Better to change the tests and explicitly pass pepper values in.